runtime/src/lib.rs

```diff
@@ -208,7 +208,7 @@ parameter_types! {
     pub const TransactionBaseFee: Balance = 100 * MICRO_RAD;
     pub const TransactionByteFee: Balance = 1 * MICRO_RAD;
 	// setting this to zero will disable the weight fee.
-	pub const WeightFeeCoefficient: Balance = 1_000;
+	pub const WeightFeeCoefficient: Balance = 100_000_000;
```

Comment:
we missed this as part of the fee PR?

Reply:
No, that is a different fee component. The fee PR only changed the authorization check of changes to state rent. This here is adjusting how much RAD each weight unit of an extrinsic will cost.